tests/lib/rules/no-missing-import.js

```diff
@@ -87,6 +87,18 @@ ruleTester.run("no-missing-import", rule, {
             filename: fixture("test.js"),
             code: "import mocha from 'mocha';",
         },
+        {
+            filename: fixture("test.js"),
+            code: "import something from 'cjs-module-with-no-main';",
+        },
+        {
+            filename: fixture("test.js"),
+            code: "import something from 'esm-module';",
+        },
+        {
+            filename: fixture("test.js"),
+            code: "import something from 'esm-module/sub';",
+        },
```

Comment:
Just to confirm: is the `import()` supported too?

Reply:
Yes. It goes through the exact same mechanism. I'll add a test, just for documentation purposes.